putData.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 set -e
 
+echo Putting data for $1 to wikibase.dev
```

Comment:
Should we use quotes?
```suggestion
echo "Putting data for $1 to wikibase.dev"
```